docx_editor_mcp/server.py

```
@@ -0,0 +1,59 @@
+
+    def shutdown(self) -> None:
+        """Gracefully shutdown server, saving dirty documents.
+
+        Best-effort: continues even if individual saves fail.
+        """
+        for cached_doc in list(self.cache.all()):
+            if cached_doc.dirty:
+                try:
+                    cached_doc.document.save()
+                    logger.info(f"Saved dirty document: {cached_doc.path}")
+                except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)